README.md

```diff
@@ -1,6 +1,6 @@
 # Doctrine Event Manager
 
-[![Build Status](https://travis-ci.org/doctrine/event-manager.svg)](https://travis-ci.org/doctrine/event-manager)
+[![Build Status](https://github.com/doctrine/event-manager/workflows/Continuous%20Integration/badge.svg)](https://github.com/doctrine/event-manager/actions)
```

Comment:
Note that this badge will only show the status of the "Continuous Integration" workflow, not the other two.

Reply:
Yup, perfect 👍